src/components/Settings.js

```diff
@@ -250,7 +250,7 @@ class SettingsModal extends React.Component {
   openFileDialogRefSetting = (event) => {
     dialog.showOpenDialog(getCurrentWindow(), {
         properties: ['openDirectory'],
-        filters: [{ name: 'USFM Files', extensions: ['usfm'] }],
+        filters: [{ name: 'USFM Files', extensions: ['*'] }],
```

Comment:
Based on files I have seen in the wild, USFM files need not necessarily .usfm extension. In fact, we have some code written to figure this out by looking at the first few lines in the file. So we should allow the user to select files with any extension.